src/worker.py

```
@@ -7101,6 +7107,162 @@ async def api_get_thread_messages(req, env, thread_id: str):
+        return err("Only jpg, jpeg, png, and webp images are allowed")
+
+    if not image_data:
+        return err("image_data is required")
+
+    # Strip data-URL prefix if the browser included it
+    if "," in image_data:
+        image_data = image_data.split(",", 1)[1]
+
+    try:
+        img_bytes = base64.b64decode(image_data)
+    except Exception:
```

Ruff (0.16.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)